cmd/vela-worker/client.go

```diff
@@ -6,18 +6,23 @@ package main
 
 import (
 	"github.com/go-vela/sdk-go/vela"
+	"github.com/hashicorp/go-retryablehttp"
 
 	"github.com/sirupsen/logrus"
 )
 
 // helper function to setup the queue from the CLI arguments.
 func setupClient(s *Server) (*vela.Client, error) {
 	logrus.Debug("creating vela client from worker configuration")
+	// create client to retryable http calls
+	retryClient := retryablehttp.NewClient()
+	// set logger to nil to avoid spam
+	retryClient.Logger = nil
```

Comment:
This seemed like the safest approach since the default behavior is to log **all** http requests regardless of success or failure.

Reply:
I think I'm okay with it for now to see how it behaves, took me awhile to find [this](https://github.com/hashicorp/go-retryablehttp/blob/master/client.go#L47-L50) so I'm adding it here:
```go
// Default retry configuration
defaultRetryWaitMin = 1 * time.Second
defaultRetryWaitMax = 30 * time.Second
defaultRetryMax     = 4
```